### ISSUE TITLE
should we use it as an API for other Go programs?

### ISSUE THREAD

**Issue by cuducos** [ellisonleao/vl#10]
I was thinking about using `vl` together with [Minha Receita](https://github.com/cuducos/minha-receita) to check the URLs before downloading them, something like this pseudo-code

```go
import github.com/npxbr/verify-links/vl

…

func Download(urls []strings) {
    err := vl.Verify(urls)
    if err != nil {
        fmt.Println(err)
        os.Exit(1)
    }

    // …
}
```

Is this interesting for this project?

**Comment by ellisonleao** [comment]
definitely! the main goal was to provide a quick cli usage for a personal project, but we can definitely create an API that can used in other codebases
